Jenkinsfile

```diff
@@ -325,24 +466,30 @@ pass = ${OBS_CREDENTIALS_PASSWORD}
                 }
             }
         }
-        stage('test') {
-            when {
-                expression { return params.PUBLISH && params.TEST }
-            }
-            steps {
-                script {
-                    def files = findFiles(glob: 'scf-*.linux-amd64*.zip')
-                    build(
-                        job: 'scf-test',
-                        parameters: [
-                            string(name: 'CONFIG_NAME', value: distPrefix() + files[0].path),
-                            string(name: 'CONFIG_PATH', value: 'https://s3.amazonaws.com/cf-opensusefs2/scf/config/' + distSubDir()),
-                            booleanParam(name: 'RUN_CATS', value: params.TEST_CATS),
-                        ],
-                        wait: false,
-                    )
-                }
-            }
+    }
+
+    post {
+        always {
+            sh """#!/bin/bash
+            set -o xtrace
+            if kubectl get storageclass ${jobBaseName()}-${BUILD_NUMBER}-hostpath ; then
```

Comment:
I think this is from a previous buggy change. The storage class name no longer depends on the job name / build number.

Reply:
Changed to just hostpath/hostpath